builtin/settingtypes.txt

```diff
@@ -867,7 +867,7 @@ menu_clouds (Clouds in menu) bool true
 #    This will smooth over some of the rough edges, and blend
 #    pixels when scaling down, at the cost of blurring some
 #    edge pixels when images are scaled by non-integer sizes.
-gui_scaling (GUI scaling) float 1.0 0.001
+gui_scaling (GUI scaling) float 1.0 0.5
```

Comment:
should probably have a sensible max, like 20 (that's almost 6x a retina display)